index.js

```diff
@@ -8,20 +8,21 @@
 'use strict'
 
 const prepack = require('prepack')
-const utils = require('rollup-pluginutils')
 
 module.exports = function rollupPluginPrepack (options) {
-  options = Object.assign({ include: '**/*.js' }, options)
-
-  const filter = utils.createFilter(options.include, options.exclude)
-  const handle = (str) => `export default ${JSON.stringify(str.trim())}`
+  const handle = (str) => `export default ${str.trim()}`
```

Comment:
Isn't more sure with JSON.stringify? I've seen in it couple of other plugins. And still think that it make sense.

Reply:
> Isn't more sure with JSON.stringify? I've seen in it couple of other plugins. And still think that it make sense.

Sure, We can use `JSON.stringify`